benchmarks/media-streaming/client/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM ubuntu:14.04
```

Comment:
Please update this to the latest version as well.

Reply:
`videoperf` which we are using is an out dated repository which can't be compiled using ubuntu 18.04 default build tools.
Instead `httperf` could be used on `ubuntu 18.04` but our benchmark depends on several tools from `videoperf`.
@sid1994 and I decided to leave it there for now. We came up with that @sid1994 will contact the author of `videoperf` to clear things.